client/components/purchase-detail/style.scss

```diff
@@ -125,3 +125,23 @@
 	color: darken( $gray, 20% );
 	font-size: 21px;
 }
+
+.purchase-detail__info {
+	flex: 1;
+	margin-left: 8px;
+	margin-top: 0;
+	max-width: 260px;
+	display: inline-block;
+
+	.purchase-detail__info-icon-container {
```

Comment:
Unless you're having trouble overriding a default style `.purchase-detail__info-icon-container` doesn't need to be nested in `.purchase-detail__info`